build.gradle

```diff
@@ -81,5 +81,7 @@ subprojects {
         theme 'mocha-parallel'
         showStandardStreams false
     }
+
+    tasks.withType(Javadoc).all { enabled = false }
```

Comment:
what is this about ?

Reply:
removed, the java doc is not working in my side previously.
Test again, it should works now.